### PR TITLE
Support for .NET Framework 4.8.1 & .NET 7 

### PR DESCRIPTION
**Pull request by Wagnerp**
* Implement #1555

### CODE REVIEW

File: PackageExplorer/PackageViewer.xaml.cs [hunk 1]
```diff
@@ -30,22 +30,34 @@ public partial class PackageViewer : UserControl
...
                     {
+                        // .NET 5 References
                         "v5.0","net5.0",
                         "v5.0-android", "net5.0-android",
                         "v5.0-ios", "net5.0-ios",
                         "v5.0-macos", "net5.0-macos",
                         "v5.0-tvos", "net5.0-tvos",
                         "v5.0-watchos", "net5.0-watchos",
-                        "v5.0-windows", "net5.0-windows",
+                        "v5.0-windows", "net5.0-windows7.0",
```
Comment (clairernovotny) [inline PackageExplorer/PackageViewer.xaml.cs:40]: Why is the 7.0 being added to windows here?

**Comment by clairernovotny** [comment]
Looks good overall except for adding the `7.0` to `windows` as per the comment above. not sure why?

**Review by clairernovotny — CHANGES_REQUESTED** [review]
Please change `windows7.0` back to `windows`

**Comment by Wagnerp** [comment]
> Please change `windows7.0` back to `windows`

@clairernovotny Reinstated for `.NET 5`, _but_ you can't do the same with >= .NET 6, otherwise it'll fail to save the package. See #1566 for details.

**Comment by Wagnerp** [comment]
Update, removed `-windows7.0` -> `-windows`

**Comment by Wagnerp** [comment]
Added .NET 8 to the list

**Pull request merged**